core/server/users.js

```diff
@@ -152,5 +152,8 @@ Meteor.users.find({ 'status.online': true }).observeChanges({
   },
   removed(id) {
     Meteor.users.update(id, { $set: { 'status.lastLogoutAt': new Date() } });
+    const user = Meteor.users.findOne(id);
```

Comment:
Not a blocker but it feels weird to fetch the candidate after the update rather than before. :)